graphics/nxwm/src/ckeyboard.cxx

```diff
@@ -167,7 +167,7 @@ int CKeyboard::open(void)
     {
       // Try to open the keyboard device
 
-      fd = std::open(CONFIG_NXWM_KEYBOARD_DEVPATH, O_RDONLY);
+      fd = open(CONFIG_NXWM_KEYBOARD_DEVPATH, O_RDONLY);
```

Comment:
```suggestion
      fd = ::open(CONFIG_NXWM_KEYBOARD_DEVPATH, O_RDONLY);
```

Reply:
Since other global function doesn't add ::, I prefer to keep the current change.